rmw_connext_cpp/src/functions.cpp

```diff
@@ -214,6 +213,57 @@ rmw_destroy_node(rmw_node_t * node)
   return destroy_node(rti_connext_identifier, node);
 }
 
+bool
+__process_topic_name(
```

Comment:
nitpick, but I think this file uses only one underscore for "non-public" methods.